src/operator/contrib/ctc_loss.cc

```diff
@@ -35,7 +35,7 @@ ctcStatus_t compute_ctc_cost(const Tensor<cpu, 3, DType> activations,
                              void *workspace, int train) {
   int minibatch = static_cast<int>(activations.size(1));
   int alphabet_size = static_cast<int>(activations.size(2));
-  int blank_label = 0;
+  int blank_label = alphabet_size-1;
```

Comment:
Are you sure? Shouldn't it be alphabet_size?

Reply:
Yes. Alphabet size may be a misnomer, because the dimension of data is *alphabet_size + 1*, and this dimension is passed in as `alphabet_size`

Reply:
On gluon/loss.py line 330, the new change specifies the following, which is inconsistent and should be corrected. Yes. It is better that alphabet_size includes blank label (with the id of alphabet_size-1).

+        input has shape `(sequence_length, batch_size, alphabet_size+1)`
 +        Note that the last dimension with index `alphabet_size` is reserved for special
 +        blank character.

Reply:
Thanks, @zhiheng-huang. I will make an example like tf doc did to clarify.